src/main/java/com/microsoft/aad/msal4j/IAccount.java

```diff
@@ -26,4 +26,12 @@
      * @return account username
      */
     String username();
+
+    /**
+     * Map of {@link ITenantProfile} objects related to this account. The keys of the map are tenant ID values, which
+     * are taken from {@link AccountCacheEntity#realm}
```

Comment:
AccountCacheEntity is not public  class - should not be referenced in java doc

Reply:
Fixed in the latest commit